mozilla-release/mobile/android/base/AndroidManifest.xml.in

```diff
@@ -6,9 +6,6 @@
       android:installLocation="internalOnly"
       android:versionCode="@ANDROID_VERSION_CODE@"
       android:versionName="@MOZ_APP_VERSION@"
-#ifdef MOZ_ANDROID_SHARED_ID
-      android:sharedUserId="@MOZ_ANDROID_SHARED_ID@"
-#endif
```

Comment:
What is this?

Reply:
And why did you remove it?

Reply:
https://developer.android.com/guide/topics/manifest/manifest-element#uid
Old ghostery browser doesnt have any id, and it can be set only once

Reply:
So we have to remove it to update the app